modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,75 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require certain BIOS settings to be configured before the host can be provisioned.
+
+.Procedure
+. Set the *UEFI/BIOS Boot Mode* to `UEFI` and ensure that the same setting is applied in the `BareMetalHost` CRD.
+
+. Set *Hard drive first* in the host boot sequence order.
++
+[NOTE]
+====
+If `rootDeviceHints` is specified in the `BareMetalHost` CRD, the hard disk must match the hard disk that you configure in the host boot sequence order.
+====
+
+. Apply the BIOS configurations for your hardware. The following table describes a representative BIOS configuration for an Intel® Xeon Skylake or Intel Cascade Lake server, based on the Intel FlexRAN 4G and 5G baseband PHY reference design.
++
+[NOTE]
+====
+The exact BIOS configuration will depend on your specific hardware and network requirements.
+====
++
+.Sample BIOS configuration for an Intel Xeon Skylake or Cascade Lake server
+[cols=2*, width="90%", options="header"]
+|====
+|BIOS Setting
+|Configuration
+
+|CPU Power and Performance Policy
+|Performance
+
+|Uncore Frequency Scaling
+|Disabled
+
+|Performance P-limit
+|Disabled
+
+|Enhanced Intel SpeedStep (R) Tech
+|Enabled
+
+|Intel Configurable TDP
+|Enabled
+
+|Configurable TDP Level
+|Level 2
+
+|Intel(R) Turbo Boost Technology
+|Enabled
+
+|Energey Efficient Turbo
+|Disabled
+
+|Hardware P-States
+|Disabled
+
+
+|Package C-State
+|C0/C1 state
```

Comment:
As someone with no context, I can't tell if this one needs more explanation.

Reply:
It doesn't, or at least that is the guidance from QE/SME, the table is a general reference, but the exact config is outside the scope of this for now. 4.9 will have a automated BIOS configuration feature.